cranelift-module/src/module.rs

```diff
@@ -716,6 +716,16 @@ where
         self.backend.isa()
     }
 
+    /// Return a reference to the backend
+    pub fn backend(&self) -> &B {
+        &self.backend
+    }
+
+    /// Return a mutable reference to the backend
+    pub fn backend_mut(&mut self) -> &mut B {
+        &mut self.backend
```

Comment:
This makes it possible to call publish before finalizing a function, which will cause simplejit to tigger SIGSEGV while trying to perform relocations after having marked the code as readonly.